.pylintrc

```diff
@@ -187,7 +187,7 @@ max-nested-blocks=5
 [FORMAT]
 
 # Maximum number of characters on a single line.
-max-line-length=80
+max-line-length=79
```

Comment:
The max-line-length default value is 80 and it **should** be 80.. Why would 79 be a better option than the default one? omg omg :-)